vms/proposervm/block.go

```diff
@@ -366,15 +375,17 @@ func (p *postForkCommonComponents) verifyPostDurangoBlockDelay(
 	var (
 		blkTimestamp = blk.Timestamp()
 		blkHeight    = blk.Height()
-		currentSlot  = proposer.TimeToSlot(parentTimestamp, blkTimestamp)
 		proposerID   = blk.Proposer()
 	)
+	// populate the slot for the block.
+	p.slot = proposer.TimeToSlot(parentTimestamp, blkTimestamp)
```

Comment:
`p *postForkCommonComponents` is actually the parent block here - I think we should probably move the `slot` definition to the `postForkBlock` type. Then we'd have:
```suggestion
	// populate the slot for the block.
	blk.slot = proposer.TimeToSlot(parentTimestamp, blkTimestamp)
```

Reply:
thanks!